pysteps/visualization/thunderstorms.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+pysteps.visualization.tstorm
+============================
+
+Methods for plotting thunderstorm cells.
+
+Created on Wed Nov  4 11:09:44 2020
+
+@author: mfeldman
+
+.. autosummary::
+    :toctree: ../generated/
+
+    plot_track
+    plot_cart_contour
+"""
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+
+try:
+    import pyproj
+
+    PYPROJ_IMPORTED = True
+except ImportError:
+    PYPROJ_IMPORTED = False
+
+from pysteps.visualization import plot_precip_field
+
+
+def plot_track(track_list, geodata=None):
+    """
```

Comment:
Shouldn't the function also contain xsize, ysize, poix and poiy, or is this part of 'geodata'?

Reply:
Well spotted, I need to update the docstring of this function!

Reply:
eaa2813